picard/script/functions.py

```diff
@@ -492,7 +492,7 @@ def func_trim(parser, text, char=None):
 
 
 @script_function(documentation=N_(
-    """`$add(x,y,*args)`
+    """`$add(x,y...)`
```

Comment:
In those functions, two arguments are mandatory, so `$add(x,y,...)` would be more correct.

`$add(x,y,...)` but `if2(a1,...)`

Reply:
If we say `$add(x,y,...)` a naive user might put `$add(x,y,)`.

`$if2` should probably be `$if2(a1,a2...)`

Reply:
I'd prefer the trailing comma, it just looks more "correct" to me, at least that is how I am used to see such notations. But no really strong feelings about it, either way is an improvement.

Reply:
Actually I sort of agree - it looks better when it is `$add(x,y...)` - I just worry that non-technical folks who are not used to programming will end up including a trailing comma. I suspect that `$mul(2,3)` and `$mul(2,3,)` may have different results as currently coded - perhaps we should correct this so that empty arguments are ignored and not treated as

Reply:
Just tested:
```
$set(_test1,a$mul(2,3))
$set(_test2,a$mul(2,3,))
_test1 = a6
_test2 = a
```
I think that mathematical functions should:
1. Ignore (trailing?) empty arguments; and
2. Return `0` rather than nothing.